src/OrleansCodeGenerator/RoslynCodeGenerator.cs

```diff
@@ -89,8 +89,14 @@ public void GenerateAndLoadForAssemblies(params Assembly[] inputs)
             }
 
             var timer = Stopwatch.StartNew();
+            var emitDebugSymbols = false;
             foreach (var input in inputs)
             {
+                if (!emitDebugSymbols)
+                {
+                    emitDebugSymbols = RuntimeVersion.IsAssemblyDebugBuild(input);
```

Comment:
Oh, sorry, you are right, I did miss the if statement... I thought there was none.